CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,117 @@
+Hello! We are excited that you want to learn more about our project contributor ladder! This contributor ladder outlines the different contributor roles within the project, along with the responsibilities and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows. Our project members are happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of things. "Responsibilities" are things that a contributor is expected to do. "Requirements" are qualifications a person needs to meet to be in that role, and "Privileges" are things contributors on that level are entitled to.
+
+## Contributor
+
+Description: A Contributor contributes directly to the project and adds value to it. Contributions doesn't only mean code contributions. Other types of contributions are valid, such as documentation improvements or community involvement as an example. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+- Responsibilities include:
+  - Follow the CNCF CoC
+  - Follow the project contributing guide
+- Requirements (one or several of the below):
+  - Report and sometimes resolve issues
+  - Occasionally submit PRs
+  - Contribute to the documentation
+  - Show up at meetings, takes notes
+  - Answer questions from other community members
+  - Submit feedback on issues and PRs
+  - Test releases and patches and submit reviews
+  - Run or helps run events
+  - Promote the project in public
+  - Help run the project infrastructure
+- Privileges:
+  - Invitations to contributor events
+  - Shout outs in the project newsletter and milestone blog post announcements
+  - Eligible to become an Organization Member
+
+## Reviewer
+
+Description: A Reviewer has responsibility for specific code, documentation, test, or other project areas. They are collectively responsible, with other Reviewers, for reviewing all changes to those areas and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+
+Reviewers are responsible for a "specific area." This can be a specific code directory, driver, chapter of the docs, test job, event, or other clearly-defined project component that is smaller than an entire repository or subproject. Most often it is one or a set of directories in one or more Git repositories. The "specific area" below refers to this area of responsibility.
+
+Reviewers have all the rights and responsibilities of an Organization Member, plus:
+
+- Responsibilities include:
+  - Following the reviewing guide
```

Comment:
is the reviewing guide from the CNCF?

Reply:
It is and there is also a '[reviewing.md](https://github.com/cncf/project-template/blob/main/REVIEWING.md)' in the template repo. I'll go through it and add one to the copa repo. :)